airdrop/application/services/user_registration_services.py

```diff
@@ -9,6 +9,56 @@
 
 class UserRegistrationServices:
 
+    def eligibility(self, inputs):
+
+        status = HTTPStatus.BAD_REQUEST
+
+        try:
+            schema = {
+                "type": "object",
+                "properties": {
+                    "airdrop_window_id": {"type": "string"},
+                    "airdrop_id": {"type": "string"},
+                    "address": {"type": "string"},
+                    "signature": {"type": "string"},
+                },
+                "required": ["signature", "address", "airdrop_id", "airdrop_window_id"],
+            }
+
+            validate(instance=inputs, schema=schema)
+
+            airdrop_id = inputs["airdrop_id"]
+            airdrop_window_id = inputs["airdrop_window_id"]
+            address = inputs["address"].lower()
```

Comment:
checkSum is what we should be using no  ?

Reply:
We are doing the checksum conversion explicitly using with utils class